.github/workflows/submitty_ci.yml

```diff
@@ -407,6 +408,10 @@ jobs:
             run: |
               cd $SUBMITTY_REPOSITORY
               sudo -E env "PATH=$PATH" bash .setup/testing/setup.sh
+              sudo jq '. |= . + {"verified_submitty_admin_user": "submitty-admin"}' /usr/local/submitty/config/submitty_users.json \
+                | sudo sponge /usr/local/submitty/config/submitty_users.json
```

Comment:
Is there a reason the `sponge` command is being used here? I haven't run the full command locally but could you instead use something like `sudo tee --append <file>` instead ?

That way we could avoid installing `moreutils` in CI

Reply:
As `jq` reads all the bytes in `submitty_users.json`, if the script contains something like 

```bash
jq 'PATTERN' submitty_users.json > submitty_users.json
```

the file will be cleared first, before `jq`.

The `sponge` here merely enables `jq` can process the content first — after receiving all data from pipe, it then opens the file and writes the data.

If this really matters, I could use something similar:

```bash
tmp=$(mktemp)
sudo jq 'PATTERN' submitty_users.json > ${tmp}
sudo mv -fv ${tmp} submitty_users.json
```

Reply:
Oh gotcha, didn't realize that order was required - that makes sense to me how you did it then